sonar-scm-git-plugin/src/test/java/org/sonarsource/scm/git/GitScmProviderTest.java

```diff
@@ -236,6 +236,29 @@ public void branchChangedLines_should_be_correct_when_change_is_not_committed()
     assertThat(changedLines).containsExactly(entry(filePath, new HashSet<>(Arrays.asList(1, 4))));
   }
 
+  @Test
+  public void branchChangedLines_should_be_correct_when_project_is_not_in_git_root() throws GitAPIException, IOException {
```

Comment:
From this name it's not clear what "correct" means. How about: `branchChangedLines_uses_relative_paths_from_project_root` or `branchChangedLines_uses_relative_paths_from_project_root_when_below_git_root`

Reply:
fixed